<?xml version="1.0"?>
<launch>
  <node name="octomap_server" type="octomap_server_node" pkg="octomap_server">
    <remap from="cloud_in" to="/camera/depth/points"/>
  </node>
</launch>